src/main/java/org/spongepowered/common/interfaces/IMixinWorld.java

```diff
@@ -100,4 +103,12 @@
 
     void notifyBlockOfStateChange(BlockPos notifyPos, Block sourceBlock, BlockPos sourcePos);
 
+    @Nullable
+    EntityPlayer getClosestPlayerToEntityWhoAffectsSpawning(net.minecraft.entity.Entity entity, double distance);
+
+    @Nullable
+    EntityPlayer getClosestPlayerWhoAffectsSpawning(double x, double y, double z, double distance);
+
+    boolean isAnyPlayerWithinRangeAtWhoAffectsSpawning(double x, double y, double z, double range);
```

Comment:
It is a "who affects spawning" version of `net/minecraft/world/World.isAnyPlayerWithinRangeAt
`, so yes.